### DISCUSSION TITLE
Add support for docker-compose profiles

### DISCUSSION THREAD

**Discussion by ghost** [testcontainers/testcontainers-node#290] [Ideas]
Would be nice to have this feature https://docs.docker.com/compose/profiles/

**Comment by cristianrgreco** [comment]
@pavlogs sure, it's available in [7.23.0](https://github.com/testcontainers/testcontainers-node/releases/tag/v7.23.0)

**Reply by ghost** [reply]
Cool. Thanks a lot 🙌

**Reply by ghost** [reply]
@cristianrgreco  Seems like there is an issue with docker-compose down.
For some reason function does not stop and remove services which has profiles
Similar to `docker-compose down `command but no error like `failed to remove network...` appeared during execution

**Reply by ghost** [reply]
This [PR](https://github.com/testcontainers/testcontainers-node/pull/300/files) should fix an issue

**Reply by cristianrgreco** [reply]
thank you @pavlogs, your fix has been released in [7.23.1](https://github.com/testcontainers/testcontainers-node/releases/tag/v7.23.1)

**Reply by ghost** [reply]
Thank you again. 🙏
Works smooth now
